src/main/model/CurlCommand.java

```diff
@@ -0,0 +1,95 @@
+package main.model;
+/* Model for the curl command: \"curl -X 'GET' \
+       'https://financial-integration.service.cnqr.tech/fi/v1/companies/1/transactiontypes/expense/transactions?limit=0&docId=%s&ignoreDocumentStatus=yes' \
+       -H 'accept: application/json' \
+       -H 'documentFormatAs: json' \
+      -H 'Authorization: Bearer $JWT'\" \
+       > ~/Downloads/%s.json;
+*/
+public class CurlCommand {
+    private String method;
+    private String url;
+    private String ignoreDocStatus;
+    private String  accept;
+    private String documentFormat;
+    private String authorization;
+    private String destination;
```

Comment:
Destination for... what? If I can't read the sample curl command, what would I assume this is? Would be good to have a more descriptive name.